workflow/rules/table.smk

```diff
@@ -1,69 +0,0 @@
-from pathlib import Path
```

Comment:
Is it intentional, that this file is just emptied, but not altogether deleted? At least it looks like that to me...

Reply:
Interesting, yes, this can be removed completely (along with the import in the Snakefile if it's still there)